plugins/udp/udp_output.go

```diff
@@ -22,6 +22,11 @@ import (
 	"runtime"
 )
 
+const (
```

Comment:
Don't really need this since it is only referenced in one place (could just put the value there).